playbook.yml

```diff
@@ -43,6 +43,16 @@
       
       - include_role:
           name: backpack
+        when: not metadata_targeted | default('true') | bool
```

Comment:
So if I can grok this ansible check correctly, this would mean the role would be included by default? Wouldn't we want the default to be false ?

Reply:
I am also not tracking this... `not metadata_targeted` would make me think false, but if it isn't set, it defaults to `true`?

Reply:
so we only want to run the role when metadata_targeted is false. the init container is outside of the backpack role. that help some?